.gitignore

```diff
@@ -40,6 +40,7 @@ __pycache__
 
 # Distribution / packaging
 .Python
+.idea/
```

Comment:
`.idea` is already part of the gitignore in the current master.

Reply:
Oh, sorry I didn't know.